src/Client/CampaignProfileHistoryClient.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Scn\EvalancheReportingApiConnector\Client;
+
+use Psr\Http\Message\RequestFactoryInterface;
+use Scn\EvalancheReportingApiConnector\EvalancheConfigInterface;
+
+final class CampaignProfileHistoryClient extends AbstractClient
+{
+    private ?int $campaignId;
```

Comment:
getCampaignProfileHistory in EvalancheConnection wants explicitly a integer, so this should be int or am I wrong?

Reply:
Aww thanks, I made a wrong assumption